src/mugs.js

```diff
@@ -734,45 +734,23 @@ define([
         tagName: 'upload',
         icon: 'fcc fcc-fd-audio-capture',
         isODKOnly: true,
+        mediaType: "audio/*", /* */
         canOutputValue: false,
         writeCustomXML: function (xmlWriter, mug) {
-            var mediaType = mug.p.mediaType;
-            if (mediaType) {
-                xmlWriter.writeAttributeString("mediatype", mediaType);
-            }
-        },
-        init: function (mug, form) {
-            mug.p.mediaType = "audio/*"; /* */
+            xmlWriter.writeAttributeString("mediatype", mug.options.mediaType);
         },
-        spec: {
-            mediaType: {
-                lstring: 'Media Type',
-                visibility: 'visible',
-                presence: 'required'
-            }
```

Comment:
Strangely, this property was not visible when I tested it, and I'm not sure why. However, the parser [does not support anything other than `audio/*`, `image/*`, and `video/*`](https://github.com/dimagi/Vellum/blob/160006-image-type-change/src/parser.js#L447-L462). It seems fine to move this to mug options (which are static) since setting custom media type values would cause the form to be un-parse-able.